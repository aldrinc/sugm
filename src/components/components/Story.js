import React from 'react';

import icon_1 from '../../images/fwc.png'; 
import icon_2 from '../../images/er.png'; 
import icon_3 from '../../images/hh.png'; 
import icon_4 from '../../images/sc.png'; 
 
 
 
class  Story extends React.Component {
 
  render() {
    return (
<div className="text_cnt">
  <div className="container-fluid">
    <div className="row">
      <div className="col-md-5">
        <div className="over_story_cnt">
          <h2>Our Story and Promise</h2>
          <p>Vincent & Bosco is a modern art brand dedicated to promoting premium art goods for artists young and old. Inspired by aspiring artists' struggles, our goal is to equip new artists with the tools they need to stand out from the crowd. In addition to supporting the arts, we're committed to operational excellence through delivering product to your door quickly and with little hassle. </p>
          <ul>
            <li> <img src={icon_1} alt="" />
              <p>Free Worldwide Shipping</p>
            </li>
            <li> <img src={icon_2}alt="" />
              <p>Easy Returns</p>
            </li>
            <li> <img src={icon_3} alt="" />
              <p>Human Helpdesk</p>
            </li>
            <li> <img src={icon_4} alt="" />
              <p>Secure Checkout</p>
            </li>
          </ul>
        </div>
      </div>
      <div className="col-md-7 hide_mob">
        <div className="holyday_cnt">
          <div className="holyday_text_cnt">
            <h3>Modern art meets modern discounts.</h3>
            <p>Free shipping for customers who purchase over $35 in goods!</p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
);
  }
}
export default (Story); 