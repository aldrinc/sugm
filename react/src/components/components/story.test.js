import React from 'react';
import Story from './Story';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
      <Story/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });

