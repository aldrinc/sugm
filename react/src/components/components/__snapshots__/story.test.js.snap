// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="text_cnt"
>
  <div
    className="container-fluid"
  >
    <div
      className="row"
    >
      <div
        className="col-md-5"
      >
        <div
          className="over_story_cnt"
        >
          <h2>
            Our Story and Promise
          </h2>
          <p>
            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industryâ€™s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
          </p>
          <ul>
            <li>
               
              <img
                alt=""
                src="fwc.png"
              />
              <p>
                Free Worldwide Shipping
              </p>
            </li>
            <li>
               
              <img
                alt=""
                src="er.png"
              />
              <p>
                Easy Returns
              </p>
            </li>
            <li>
               
              <img
                alt=""
                src="hh.png"
              />
              <p>
                Human Helpdesk
              </p>
            </li>
            <li>
               
              <img
                alt=""
                src="sc.png"
              />
              <p>
                Secure Checkout
              </p>
            </li>
          </ul>
        </div>
      </div>
      <div
        className="col-md-7 hide_mob"
      >
        <div
          className="holyday_cnt"
        >
          <div
            className="holyday_text_cnt"
          >
            <h3>
              Happy Holidays!
            </h3>
            <p>
              In the spirit of the holidays here's 15% OFF our art collection! Use code ARTSY at checkout
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
